crates/common/sync/src/lib.rs

```diff
@@ -0,0 +1,148 @@
+use alloy_primitives::B256;
+use anyhow::{anyhow, ensure};
+use checkpoint::fetch_default_checkpoint_url;
+use ream_consensus::{
+    blob_sidecar::{BlobIdentifier, BlobSidecar},
+    constants::SECONDS_PER_SLOT,
+    electra::{beacon_block::SignedBeaconBlock, beacon_state::BeaconState},
+    execution_engine::rpc_types::get_blobs::BlobAndProofV1,
+    misc::{compute_epoch_at_slot, compute_start_slot_at_epoch},
+};
+use ream_fork_choice::{
+    handlers::on_tick,
+    store::{Store, get_forkchoice_store},
+};
+use ream_network_spec::networks::network_spec;
+use ream_rpc::types::response::BeaconVersionedResponse;
+use ream_storage::{db::ReamDB, tables::Table};
+use tracing::{info, warn};
+pub mod checkpoint;
```

Comment:
```suggestion
pub mod checkpoint;

use alloy_primitives::B256;
use anyhow::{anyhow, ensure};
use checkpoint::fetch_default_checkpoint_url;
use ream_consensus::{
    blob_sidecar::{BlobIdentifier, BlobSidecar},
    constants::SECONDS_PER_SLOT,
    electra::{beacon_block::SignedBeaconBlock, beacon_state::BeaconState},
    execution_engine::rpc_types::get_blobs::BlobAndProofV1,
    misc::{compute_epoch_at_slot, compute_start_slot_at_epoch},
};
use ream_fork_choice::{
    handlers::on_tick,
    store::{Store, get_forkchoice_store},
};
use ream_network_spec::networks::network_spec;
use ream_rpc::types::response::BeaconVersionedResponse;
use ream_storage::{db::ReamDB, tables::Table};
use tracing::{info, warn};
```

the mod should be at the top of the file